plugins/arRestApiPlugin/modules/api/actions/informationobjectsUpdateAction.class.php

```diff
@@ -30,7 +30,7 @@ protected function put($request, $payload)
 
     if (QubitInformationObject::ROOT_ID === (int)$this->io->id)
     {
-      throw new QubitApiForbiddenException;
+      throw new QubitApiForbiddenException('Forbidden');
```

Comment:
It'd be nice to have a default exception message so we don't have to give it one. If you changed `QubitApiForbiddenException` as the following:

```php
class QubitApiForbiddenException extends Exception
{
  const MESSAGE = 'Forbidden';

  const STATUS_CODE = 403;

  public function __construct($message = null, $code = 0, Exception $previous = null)
  {
    $message = is_null($message) ? self::MESSAGE : $message;

    return parent::__construct($message, $code, $previous);
  }

  public function getStatusCode()
  {
    return self::STATUS_CODE;
  }
}
```

Then `throw new QubitApiForbiddenException;` would be enough but we could still add a custom message when necessary. Does that sound like a good idea?